src/components/proofs/ProofList.tsx

```diff
@@ -19,11 +19,12 @@ export function ProofListSuspended() {
     <>
       <Scrollbar>
         <div className={space('space-y-2')}>
-          {account && (
-            <>
-              <ERC721ProofSection account={account} network={Network.Mainnet} />
-              <ERC721ProofSection account={account} network={Network.Goerli} />
-            </>
+          {Object.values(data).map(
+            ({ badgeType, network }) =>
+              badgeType === BadgeSourceType.ERC721 &&
+              account && (
```

Comment:
I think we should check the account before the mapping, so we don't try to map without connected account